README.md

```diff
@@ -122,7 +119,7 @@ Example command:
 
 ### Deployment
 
-You can deploy this project using:
+** Deployment** https://abenezerassefa.github.io/Abeni-Pro/
```

Comment:
- [ ] Good job deploying it, I would advise you to update the live demo so that it could be easily accessed from there.

![image](https://user-images.githubusercontent.com/86200356/221526906-90c8f1c5-5435-40ef-a5e6-dbd5403bf315.png)